File: QuoteQuiz.BLL/Models/AnswerModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QuoteQuiz.BLL.Models
{
    public class AnswerModel
    {
        public Guid Id { get; set; }
        public Guid? QuoteId { get; set; }
        public string AnswerText { get; set; }
        public bool IsCorrect { get; set; }
        public virtual QuoteModel Quote { get; set; }
    }
}

File: QuoteQuiz.Web/ClientApp/src/app/services/answer/models/AnswerModel.ts
import { QuoteModel } from '../../quote/models/QuoteModel';

export class AnswerModel {
    id: string;
    quoteId: string;
    answerText: string;
    quoteText: string;
    isCorrect?: boolean;
    quote: QuoteModel = new QuoteModel();
}
